packages/ApiSamples/scripts/ux/Interactivity/hints.js

```diff
@@ -13,10 +13,23 @@ $(addNCIcon).on('click', () => {
 });
 
 grok.events.onDialogShown.pipe(
-  rxjs.operators.first(),
-  rxjs.operators.filter((dlg) => dlg.title === 'Add New Column'))
+  rxjs.operators.filter((dlg) => dlg.title === 'Add New Column'),
+  rxjs.operators.first())
```

Comment:
Change the operators to make the subscription work (due to the wizard, 'Add New Column' dialog isn't the first one to open)